eZ/Bundle/EzPublishCoreBundle/Resources/config/security.yml

```diff
@@ -1,5 +1,6 @@
 parameters:
     security.authentication.provider.dao.class: eZ\Publish\Core\MVC\Symfony\Security\Authentication\RepositoryAuthenticationProvider
+    security.authentication.provider.rememberme.class: eZ\Publish\Core\MVC\Symfony\Security\Authentication\RememberMeRepositoryAuthenticationProvider
```

Comment:
Damn, this won't work in symfony 3 which we partly support in 6.12 and up.

param has been removed, see: d9ee7e0a6f44259df3613a715e11780b065e1bc8

Reply:
I'll try to adapt in merge

Reply:
@andrerom Did you solve this?

Reply:
@glye yes afaik I did, should be this one I think: https://github.com/ezsystems/ezpublish-kernel/commit/d0ab564019d67c6c11fb96582f25a9099e9668c7#diff-ead6105ef22c62e5fba69ab6aa931a84R47